test/integration/singlecluster/controller/jobs/job/job_controller_test.go

```diff
@@ -964,6 +964,47 @@ var _ = ginkgo.Describe("Job controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 			})
 		})
 	})
+
+	ginkgo.It("should deduplicate env variables", func() {
```

Comment:
Extend the scenario or add a new test to demonstrate what happens on eviction

Reply:
Done